### ISSUE TITLE
json-schema-form

### ISSUE THREAD

**Issue by kbzowski** [Limenius/Liform#6]
Can Liform serialize Symfony forms to json comptible with Json Schema Form (https://github.com/json-schema-form)?

**Comment by nacmartin** [comment]
The output of Liform is a valid json-schema, so that part is covered.

I see that json-schema-form can also use a separate UI-schema with UI hints. We add the ui hints directly to the Json-schema, not using two separate specifications, so that part will need extra work, to write another serializer to generate that part. It is not the path that we followed because writing another serializer more looks like a pain.

So, in short, it should work, but for getting the exact generated form that you want you may need to put extra work.
